modules/exploits/unix/webapp/ajenti_auth_username_cmd_injection.rb

```diff
@@ -0,0 +1,88 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
+
+  def initialize(info = {})
+    super(update_info(info,
+                      'Name'            => 'Ajenti auth username Command Injection',
```

Comment:
Despite what Rubocop suggests, indenting the info hash to 2 spaces is preferred.